doc/classes/AudioStreamPlayer.xml

```diff
@@ -72,7 +73,15 @@
 			If [code]true[/code], the playback is paused. You can resume it by setting [code]stream_paused[/code] to [code]false[/code].
 		</member>
 		<member name="volume_db" type="float" setter="set_volume_db" getter="get_volume_db" default="0.0">
-			Volume of sound, in dB.
+			Volume of sound, in dB. (To convert to dB, try [code]linear2db[/code].)
+		</member>
+		<member name="volume_scale_l" type="float" setter="set_volume_scale_l" getter="get_volume_scale_l" default="1.0">
+			Volume scale for left channel in linear (0.0 is silent, 1.0 is full volume) form.
+			Can be used to forcefully globally pan sounds without relying on the camera.
```

Comment:
Don't use separate lines, as this would cause them to be on separate paragraphs:

```suggestion
			Volume scale for left channel in linear (0.0 is silent, 1.0 is full volume) form. Can be used to forcefully pan sounds globally without relying on the camera.
```

Reply:
Applied in latest commit.